.github/workflows/cypress.yml

```diff
@@ -5,20 +5,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
 
       - name: set up node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v3
```

Comment:
Use the reusable cypress workflow?

Reply:
@pyphilia will it work with non-ts apps ?